Tests/src/PublicAPITests/ECommerceSystemTests.cpp

```diff
@@ -136,4 +136,45 @@ CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetProductInformationTest)
 
 	LogOut(UserSystem);
 }
-#endif
\ No newline at end of file
+#endif
+
+#if RUN_ECOMMERCE_TESTS || RUN_ECOMMERCE_GET_CHECKOUT_INFORMATION_TEST
+CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetCheckoutInformationTest)
```

Comment:
Since this is a manual test, can we add some documentation to this to explain what needs to be done before this test can be run. This could also be done for the test above also. There is a lot of setup that needs to be done that I'm currently struggling with using the internal google doc but that is not available externally. We should also note that doing the setup for the tests requires swagger usage and we currently have swagger behind the VPN so not everyone will be able to run these tests. 

Unrelated but I think that the non-happy path stuff could exist in it's own test since that could run without the manual intervention.